client/blocks/comment-detail/index.jsx

```diff
@@ -80,18 +81,40 @@ export class CommentDetail extends Component {
 
 	edit = () => noop;
 
+	getCommentObject = () => ( {
+		commentId: this.props.commentId,
+		isLiked: this.props.commentIsLiked,
+		postId: this.props.postId,
+		status: this.props.commentStatus,
+	} );
```

Comment:
What sort of comment object is this? I think keeping it here or moving it out as a normal function works, but this is likely a smell that we could be doing this translation in the data-layer or in a selector.